app/controllers/UploadController.scala

```diff
@@ -90,7 +74,7 @@ class UploadController @Inject() (
       transferAgreementStatus match {
         case Some(CompletedValue.value) =>
           uploadStatus match {
-            case Some(InProgressValue.value) =>
+            case Some(InProgressValue.value) | Some(CompletedWithIssuesValue.value) =>
```

Comment:
Will this still show an error on the page if the status is `CompletedWithIssuesValue`?

Reply:
Yes it will. This is extra case is for when the user clicks "back" on the their browser but the backend checks failed they won't see the "something went wrong page"